client/src/shared/GameHeader.tsx

```diff
@@ -70,26 +71,32 @@ function QRCodeDialog({ gameId }: QRCodeDialogProps) {
   return (
     <Dialog.Root>
       <Dialog.Trigger>
-        <Link onClick={handleShare} size="2" className="whitespace-nowrap">
-          <Text color="amber">Game:</Text> {gameId}
-          <BsShare className="ml-.5 inline text-xs" />
-        </Link>
+        <button>
+          <Flex align="center" gap="1">
```

Comment:
Do this `asChild` and make button the child.

Reply:
I am also surprised this doesn't cause other changes in the UI since buttons have a bunch of default styling.